Formula/dvtm.rb

```diff
@@ -11,8 +11,10 @@ class Dvtm < Formula
     sha256 "88bffcbc907f9ffa900331acf5f994af07f5c1787c328ac56c72935813f57b92" => :mavericks
   end
 
-  # https://github.com/martanne/dvtm/issues/19
-  depends_on MaximumMacOSRequirement => :yosemite
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/master/dvtm/patch-vt.h-compat.patch"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!